filebeat/prospector/udp/harvester.go

```diff
@@ -0,0 +1,52 @@
+package udp
+
+import (
+	"net"
+	"time"
+
+	"github.com/elastic/beats/filebeat/channel"
+	"github.com/elastic/beats/filebeat/util"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+type Harvester struct {
+	outlet channel.Outleter
+	done   chan struct{}
+}
+
+func (h *Harvester) Run() {
+	listener, err := net.ListenPacket("udp", ":8080")
+
+	if err != nil {
+		logp.Err("error happened: %s", err)
+		return
+	}
+	defer listener.Close()
+
+	//logp.Info("Now listening for logs via %s on %s", ll.config.Protocol, address)
+	buffer := make([]byte, 1000)
```

Comment:
Thanks for sharing the thoughts. I set it 1024 for the moment and made it configurable. Time will tell and we can still change the default.